diff --git a/cars/lib.py b/cars/lib.py
--- a/cars/lib.py
+++ b/cars/lib.py
@@ -28,7 +28,10 @@
         "meta-llama/Llama-3.2-1B-Instruct",
         # Qwen
         "Qwen/Qwen2.5-0.5B-Instruct",
+        "Qwen/Qwen2.5-1.5B-Instruct",
+        "Qwen/Qwen2.5-Coder-1.5B-Instruct",
         "Qwen/Qwen2.5-Coder-7B-Instruct",
+        "Qwen/Qwen2.5-Coder-14B-Instruct",
         "Qwen/Qwen2.5-7B-Instruct",
         "Qwen/Qwen2.5-14B-Instruct",
         # DeepSeek
diff --git a/mcmc/lib.py b/mcmc/lib.py
--- a/mcmc/lib.py
+++ b/mcmc/lib.py
@@ -49,7 +49,10 @@
         "meta-llama/Llama-3.2-1B-Instruct",
         # Qwen
         "Qwen/Qwen2.5-0.5B-Instruct",
+        "Qwen/Qwen2.5-1.5B-Instruct",
+        "Qwen/Qwen2.5-Coder-1.5B-Instruct",
         "Qwen/Qwen2.5-Coder-7B-Instruct",
+        "Qwen/Qwen2.5-Coder-14B-Instruct",
         "Qwen/Qwen2.5-7B-Instruct",
         "Qwen/Qwen2.5-14B-Instruct",
         # DeepSeek
