diff --git a/cars/lib.py b/cars/lib.py
--- a/cars/lib.py
+++ b/cars/lib.py
@@ -101,6 +101,7 @@ class ConstrainedModel():
             formatted_prompt = self.tokenizer.apply_chat_template(
                 messages,
                 tokenize=False,
                 add_generation_prompt=True,
+                enable_thinking=False,
             )
             assert type(formatted_prompt) == str
             return formatted_prompt
diff --git a/mcmc/lib.py b/mcmc/lib.py
--- a/mcmc/lib.py
+++ b/mcmc/lib.py
@@ -117,6 +117,7 @@ class ConstrainedModel():
             formatted_prompt = self.tokenizer.apply_chat_template(
                 messages,
                 tokenize=False,
                 add_generation_prompt=True,
+                enable_thinking=False,
             )
             assert type(formatted_prompt) == str
             return formatted_prompt
